packages/x-components/tests/e2e/cucumber/keyboard-navigation.spec.ts

```diff
@@ -11,7 +11,9 @@ type Direction = 'above' | 'below' | 'on the left' | 'on the right';
 
 // Scenario 1
 And('{string} element position is stored', (focusableElement: string) => {
```

Comment:
I've just discovered that we've an error in the following step

```cucumber
When(
  '{string} arrow is pressed {int} times',
  (direction: 'up' | 'right' | 'bottom' | 'right', pressedTimes: number) => {
    for (let i = 0; i < pressedTimes; i++) {
      cy.focused().type(`{${direction}arrow}`);
    }
  }
);
```

As you can see `right` is duplicated and left is not present, I would like to extract this 4 options to a type and also simplify the loop:

`type Move = 'up' | 'right' | 'bottom' | 'left';`

```cucumber
When('{string} arrow is pressed {int} times', (direction: Move, pressedTimes: number) => {
  Array.from({ length: pressedTimes }).forEach(() => {
    cy.focused().type(`{${direction}arrow}`);
  });
});